main.go

```diff
@@ -15,47 +17,47 @@ func printHelp() {
 		" gosplat [input dir/file] , runs directory against model\n")
 }
 
-func checkForFlags(args []string) uint8 {
-	if len(args) == 1 {
-		printHelp()
-		return earlyReturn
-	}
-	for _, arg := range args {
-		if arg == "-h" || arg == "--help" {
-			printHelp()
-			return earlyReturn
-		}
-	}
-	return continueProgram
-}
-
 var (
 	pythonPath = os.Getenv("HOME") + "/.local/share/gosplat/src/python_helper/fast_model_compare.py"
 	modelBin   = os.Getenv("HOME") + "/.local/share/gosplat/fast-fb-model.bin"
 )
 
 const (
-	earlyReturn     uint8 = 1
-	continueProgram uint8 = 2
+	earlyReturn      uint8 = 1
+	continueProgram  uint8 = 2
+	accuracyOn       uint8 = 3
+	nameSuggestionOn uint8 = 4
 )
 
-func main() {
-	if checkForFlags(os.Args) == earlyReturn {
-		return
-	}
-	dir := os.Args[1]
+// flagValues
+var (
+	dir             string
+	accuracy        int
+	help            bool
+	nameSuggestions bool
+)
+
+func checkForFlags() {
+	flag.StringVar(&dir, "d", ".", "Directory; Dir which analysis starts on")
```

Comment:
It won't be added to the generated help printout, also does not really affect the usability of the program. If you want to look in current do `gosplat` otherwise you can choose an optional directory to point it towards.

Reply:
Det låter lite fucky, kan vi iaf bara ta bort att den defaultar till current sir. Det är ju väldigt udda. Och sen ha required att man använder -d flaggan